### DISCUSSION TITLE
where should i place the gliner-spacy component in a spacy pipeline ?

### DISCUSSION THREAD

**Discussion by MedAmineDH** [theirstory/gliner-spacy#10] [Q&A]
i'm using the fr_dep_news_trf pipeline 
`gliner_spacy_pipeline = spacy.load("fr_dep_news_trf")`
['transformer', 'morphologizer', 'parser', 'attribute_ruler', 'lemmatizer']
and i'm curious where should i place the gliner-spacy NER component for a better result !

**Comment by wjbmattingly** [comment]
I would place it before the NER component to give it primacy (if that is your intention). Another option would be to disable the NER component for the trf model and place this component at the end.
